elasticdl/doc/tutorials/elasticdl_cluster.md

```diff
@@ -0,0 +1,41 @@
+# ElasticDL on cluster
+
+## Environment preparation
+
+You should install ElasticDL first. Please refer to the installation part in [elastic_local](./elasticdl_local.md) doc.
+
+Then, please build needed images.
+
+```bash
+bash elasticdl/docker/build_all.sh
+```
+
+## Submit job to cluster
+
+The submit command is similar to local mode. Following is an exmaple:
+
+
+```bash
+python -m elasticdl.python.elasticdl.client train \
+ --image_base=elasticdl:ci \
+ --cluster_spec=$CLUSTER_SPEC \
+ --model_zoo=./model_zoo \
+ --docker_image_prefix=$DOCKER_HUB_REPO \
```

Comment:
Readers might be confused about what `DOCKER_HUB_REPO` and `CLUSTER_SPEC` refers to. Perhaps add some descriptions on those